scripts/dts/python-devicetree/src/devicetree/dtlib.py

```diff
@@ -401,11 +406,13 @@ def to_strings(self):
         try:
             return self.value.decode("utf-8").split("\0")[:-1]
         except UnicodeDecodeError:
-            _err("value of property '{}' ({}) on {} in {} is not valid UTF-8"
+            _err("value of property '{}' ({!r}) on {} in {} is not valid UTF-8"
                  .format(self.name, self.value, self.node.path,
                          self.node.dt.filename))
 
-    def to_node(self):
+        assert False            # appease the type checker
```

Comment:
another possible fix (should work without the dummy assert):

```python
try:
    val = f()
except X:
    _err(...)

return val
```
```

Reply:
> another possible fix (should work without the dummy assert):

sure; done